src/components/Time/DateTimeInput.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from 'react';
+import { Tooltip, TextInput, Button, ButtonVariant, Modal } from '@patternfly/react-core';
+import { IntervalInSeconds, TimeInSeconds } from 'types/Common';
+import { toString } from './LocalTime';
+import { KialiIcon, defaultIconStyle } from 'config/KialiIcon';
+import Slider from 'components/IstioWizards/Slider/Slider';
+import { style } from 'typestyle';
+
+interface DateTimeInputProps {
+  minTime: TimeInSeconds;
+  steps?: number;
+  time: TimeInSeconds;
+  tooltip: string;
+
+  onTimeChange: (time: TimeInSeconds) => void;
+}
+
+type DateTimeInputState = {
```

Comment:
Just for consistency, this should be an interface too.

Reply:
[Even though it took me longer than I want to admit] This component will be replaced by a DateTimePicker, so no more work merited here.